### PR TITLE
Fix broken code links in comments of SetOptions.java

### PR DESCRIPTION
**Pull request by wu-hui**

### CODE REVIEW

File: firebase-firestore/src/main/java/com/google/firebase/firestore/SetOptions.java [hunk 1]
```diff
@@ -26,10 +26,10 @@
 
 /**
  * An options object that configures the behavior of {@code set()} calls. By providing one of the
- * SetOptions objects returned by {@link #merge}, {@link #mergeFields} and {@link #mergeFieldPaths},
- * the {@code set()} calls in {@link DocumentReference}, {@link WriteBatch} and {@link Transaction}
- * can be configured to perform granular merges instead of overwriting the target documents in their
- * entirety.
+ * SetOptions objects returned by {@link #merge()}, {@link #mergeFields(List<String>)} and {@link
```
Comment (vkryachko) [inline firebase-firestore/src/main/java/com/google/firebase/firestore/SetOptions.java:29]: I don't think it's valid syntax. it should be something like:
```suggestion
 * SetOptions objects returned by {@link #merge()}, {@link #mergeFields(List)} and {@link
```

i.e. without generic parameters.

**Review by vkryachko — COMMENTED** [review]
SetOptions.java is not the only file mentioned in the bug, please make sure you fix the rest of the issues.

File: firebase-firestore/src/main/java/com/google/firebase/firestore/SetOptions.java (hunk 1 above)
Comment (wu-hui) [reply firebase-firestore/src/main/java/com/google/firebase/firestore/SetOptions.java]: Done.

**Review by wu-hui — COMMENTED** [review]
The original bug does not mention, but I think the other place is DocumentId.java?

**Review by vkryachko — APPROVED** [review]
Thanks

**Pull request merged**